webtestingii/src/App.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import Display from "./Display";
+import Dashboard from "./Dashboard";
+import "./App.css";
+
+class App extends Component {
+  state = {
+    strikes: 0,
+    balls: 0,
+    outs: 0,
+    runs: 0
+  };
+
+  baseHit = () => {
+    this.setState(prevState => ({
+      balls: 0,
+      strikes: 0,
+      runs: prevState.runs + 1
```

Comment:
well, aren't we optimisitic! every hit is going to result in a run, eh? guess everybody's getting homreruns in your baseball stadium :laughing: